**GENE EXPRESSION (DOWNREGULATION AND UPREGULATION)**

**Author(@slack) : Henry Momanyi(@Henry Momanyi)**
GitHub Repo:https://github.com/Henry-momanyi/HackBio_Cancer_Internship/blob/main/HackBio_Cancer_internship.md
Gene expression entails the path followed in the transfer of unfunctional genes into a functional product. These functional products may include: RNA, DNA and proteins.(Assefa et al., 2018)&#x20;

The pathways followed involves transcription and translation.


### <a id="_Toc165448399"></a>

During this stage a complementary strand is formed which is a called a gene transcript. After transcription, post transcription modifications occur which include; Maturation off newly synthesized primary RNA, splicing where the non-coding gene regions (Introns) are removed, polyadenylation the product formed is ready for translation


### <a id="_Toc165448400"></a>

Involve the transformation of the mature transcript into a functional product

Occurs mainly in three stages which include:

<!--[if !supportLists]-->·         <!--[endif]-->Initiation

<!--[if !supportLists]-->·         <!--[endif]--> Elongation

<!--[if !supportLists]-->·         <!--[endif]-->Termination.

**Upregulation**

**The complementary process that involves increase in quantities of cellular components this in turn increase the rate at which receptors are produced increasing the immune response and cell signaling. This can be caused by chronic exposure of receptors to antagonists leading to overproduction of cellular components i.e., Cytokine storm**

**Downregulation**

This in turn involves the reduced receptor production due to reduced cell activity and signal transduction due to chronic exposure to agonists.

The expression level can be measured or determined through various ways which include comparison of the P values with normal expression level and determining the Fold change and the false discovery rate unto which the data differs from the normal data expression profiles.
Using the Enrichment profiles and analysis tools, i.e, Gene Ontology, KEGG pathway,Enrichment Atlas, the data can be mornitored and analysed which can provide key steps in in analysis,research and in determining the various associated pathways in development,proliferation and spread of cancer.

**REFERENCES**

Zheng, T., Wang, A., Hu, D., & Wang, Y. (2017). Molecular mechanisms of breast cancer metastasis by gene expression profile analysis. _Molecular Medicine Reports_, _16_(4), 4671–4677. https\://doi.org/10.3892/mmr.2017.7157

Shi, S. H., Zhang, W., Jiang, J., & Sun, L. (2017). Identification of altered pathways in breast cancer based on individualized pathway aberrance score. _Oncology Letters_, _14_(2), 1287–1294. https\://doi.org/10.3892/ol.2017.6292

Li, Y., Zhou, X., Liu, J., Yin, Y., Yuan, X., Yang, R., Wang, Q., Ji, J., & He, Q. (2020). Differentially expressed genes and key molecules of BRCA1/2-mutant breast cancer: Evidence from bioinformatics analyses. _PeerJ_, _2020_(1). https\://doi.org/10.7717/peerj.8403

Mueller, C. R., & Roskelley, C. D. (2003). Regulation of BRCA1 expression and its relationship to sporadic breast cancer. In _Breast Cancer Research_ (Vol. 5, Issue 1, pp. 45–52). https\://doi.org/10.1186/bcr557

Assefa, A. T., De Paepe, K., Everaert, C., Mestdagh, P., Thas, O., & Vandesompele, J. (2018). Differential gene expression analysis tools exhibit substandard performance for long non-coding RNA-sequencing data. _Genome Biology_, _19_(1). https\://doi.org/10.1186/s13059-018-1466-5

Alam, M. S., Sultana, A., Reza, M. S., Amanullah, M., Kabir, S. R., & Mollah, M. N. H. (2022b). Integrated bioinformatics and statistical approaches to explore molecular biomarkers for breast cancer diagnosis, prognosis and therapies. _PLoS ONE_, _17_(5 5). https\://doi.org/10.1371/journal.pone.0268967

Bagegni, N., Thomas, S., Liu, N., Luo, J., Hoog, J., Northfelt, D. W., Goetz, M. P., Forero, A., Bergqvist, M., Karen, J., Neumüller, M., Suh, E. M., Guo, Z., Vij, K., Sanati, S., Ellis, M., & Ma, C. X. (2017). Serum thymidine kinase 1 activity as a pharmacodynamic marker of cyclin-dependent kinase 4/6 inhibition in patients with early-stage breast cancer receiving neoadjuvant palbociclib. _Breast Cancer Research_, _19_(1). https\://doi.org/10.1186/s13058-017-0913-7

Land H, Parada LF, Weinberg RA (1983) Cellular oncogenes and multistep carcinogenesis. Science 222(4625):771–778

Hanahan D, Weinberg RA (2000) The hallmarks of cancer. Cell 100(1):57–70.

Hanahan D, Weinberg RA (2011) Hallmarks of cancer: The next generation. Cell 144(5):646–674.

Cancer Genome Atlas Research Network (2011) Integrated genomic analyses of ovarian carcinoma. Nature 474(7353):609–615.

Uhlen M, et al. (2010) Towards a knowledge-based Human Protein Atlas. Nat Biotechnol 28(12):1248–1250.

MacKenzie KL, et al. (2002) Multiple stages of malignant transformation of human endothelial cells modelled by co-expression of telomerase reverse transcriptase, SV40 T antigen and oncogenic N-ras. Oncogene 21(27):4200–4211

Land H, Parada LF, Weinberg RA (1983) Tumorigenic conversion of primary embryo fibroblasts requires at least two cooperating oncogenes. Nature 304(5927):596–602

Hahn WC, et al. (1999) Creation of human tumour cells with defined genetic elements. Nature 400(6743):464–468

Kahlert C, et al. (2011) Low expression of aldehyde dehydrogenase 1A1 (ALDH1A1) is a prognostic marker for poor survival in pancreatic cancer. BMC Cancer 11:275.
